history/peter-toma.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Developer of the first machine translation product
+---
+# Peter Toma
+Peter Toma was a computer scientist and linguistics researcher. 
+
+In 1956, Toma developed software for machine translation at the California Institute of Technology. 
+The system translated Russian texts into English. Later, he worked at Georgetown University where the system 
+Serna was developed for the same translation direction. 
+
+In 1962, he founded a company which created the systems Autotran and Technotran. 
+They were specialized in Russian-to-English translation in the fields of atomic energy and medicine.  
```

Comment:
They were for translating Russian to English and specialized for translating content about atomic energy and medicine.